tests/app/main/views/test_index.py

```diff
@@ -32,6 +32,24 @@ def test_security_policy_redirects_to_policy(client, url):
     assert response.location == "https://vdp.cabinetoffice.gov.uk/.well-known/security.txt"
 
 
+@pytest.mark.parametrize(
+    "url",
+    [
+        "/services/_status",
+        "/services/11111111-1111-4111-1111-111111111111/documents/22222222-2222-4222-2222-222222222222",
+        "/services/11111111-1111-4111-1111-111111111111/documents/22222222-2222-4222-2222-222222222222.pdf",
+        "/services/11111111-1111-4111-1111-111111111111/documents/22222222-2222-4222-2222-222222222222/check",
+        "/services/11111111-1111-4111-1111-111111111111/documents/22222222-2222-4222-2222-222222222222/check?key=123456",
+        "/services/11111111-1111-4111-1111-111111111111/documents/22222222-2222-4222-2222-222222222222/check?random_parameter=xyz",
+    ],
```

Comment:
The utf-8 decoding was necessary because the `query_string` attribute is a [raw bytestring](https://tedboy.github.io/flask/generated/generated/flask.Request.query_string.html#flask.Request.query_string). 

All these urls here should be urls that we have generated, i.e. not include any special characters.